docs/api/python/contrib/onnx.md

```diff
@@ -0,0 +1,111 @@
+# ONNX API
+
+## Overview
+
+The `mxnet.contrib.onnx` package refers to the APIs and interfaces that implements ONNX model format support for Apache MXNet.
+
+With ONNX format support for MXNet, developers can build and train models with PyTorch, CNTK, or Caffe2, and import these models into MXNet to run them for inference using MXNet’s highly optimized engine.
+
+```eval_rst
+.. warning:: This package contains experimental APIs and may change in the near future.
+```
+
+```eval_rst
+.. note:: **Install ONNX** which needs protobuf compiler to be installed separately. Please **follow the instructions to install ONNX** [here](https://github.com/onnx/onnx).
+```
+
+This document describes the ONNX APIs in mxnet.
+
+```eval_rst
+.. autosummary::
+    :nosignatures:
+
+    mxnet.contrib.onnx.import_model
+```
+
+## Import ONNX model into Mxnet
+
+The code demonstration in this document assumes that the following packages are installed and imported.
+
+```python
+from PIL import Image
+import numpy as np
+import mxnet as mx
+import mxnet.contrib.onnx as onnx_mxnet
+```
+
+The PIL package is a Python Image Processing package and is required for input preprocessing. It can be installed as follows
+
+```python
+pip install Pillow
+```
+
+We will now try to import a [Super_Resolution model](http://pytorch.org/tutorials/advanced/super_resolution_with_caffe2.html), trained with PyTorch, and run inference in MXNet. PyTorch provides a way to export models in ONNX protobuf format. Using this functionality, we have exported the model into ONNX format.
+
+You can download the converted ONNX model from
+[here](https://s3.amazonaws.com/onnx-mxnet/examples/super_resolution.onnx).
+
+A pre-trained model in MXNet contains two elements: a symbolic graph, containing the model's network definition, and a binary file containing the model weights. You can import the ONNX model and get the symbol and parameters objects using "import_model" API as shown below:
```

Comment:
bulleted points
* Symbolic graph(with the extension .json) - containing the model's network definition
* Binary file(with the extension .params) - containing model weights.
..